### DISCUSSION TITLE
v2024.10.1に更新すると、Failed to load画面が出てしまう

### DISCUSSION THREAD

**Discussion by mk499490** [misskey-dev/misskey#14846] [Q&A] [answered]
### 💡 Summary

![Image](https://github.com/user-attachments/assets/5f2d0e32-6042-4407-b2d9-b33d0e8c2261)

Misskeyのバージョンをv2024.10.1に更新したところ、Failed to load画面が出てしまう。

他の端末、キャッシュ削除、Cloudflareのバイパスなども試したが状況は変わらなかった。

### 🥰 Expected Behavior

タイムラインの画面ないしログイン画面が正常に表示される

### 🤬 Actual Behavior

Failed to load画面が表示される

### 📝 Steps to Reproduce

1. Docker環境下で、Misskeyを2024.8.0→2024.10.1に、[手順](https://misskey-hub.net/ja/docs/for-admin/install/guides/docker/#misskey%E3%81%AE%E3%82%A2%E3%83%83%E3%83%97%E3%83%87%E3%83%BC%E3%83%88%E6%96%B9%E6%B3%95)に従って更新する
2. Misskeyにブラウザでアクセスする

### 💻 Frontend Environment

* Model and OS of the device(s): Mac mini M1 2020 (macOS Sequoia 15.0.1)、Pixel 6a (Android 14)
* Browser: Vivaldi、Safari、Chrome
* Server URL: https://mk.peg.osaka
* Misskey: v2024.10.1

### 🛰 Backend Environment (for server admin)

* Installation Method or Hosting Service: Docker
* Misskey: v2024.10.1
* OS and Architecture: Ubuntu Server 22.04.5 LTS

### Do you want to address this bug yourself?

- [ ] Yes, I will patch the bug myself and send a pull request

**Comment by samunohito** [comment]
（ひとまず何が起こってるかだけ）
実際にアクセスしてみたところ、どうやらtsファイルを読み込もうとしている様子…
![image](https://github.com/user-attachments/assets/683ac747-0949-4556-a3af-0ed86bdee91e)
　
これは自分のサーバの物ですが、通常であれば以下のようにjsファイルへとトランスパイルされたファイルを読み込むはずです
![image](https://github.com/user-attachments/assets/79bb11b1-0cbd-4644-bbdb-4675395c31dc)

**Comment by mk499490** [comment]
他の（フォークの）リポジトリですが、関連する情報があったので記録がてら共有しておきます: [https://github.com/MisskeyIO/misskey/pull/147](https://github.com/MisskeyIO/misskey/pull/147)

**Comment by mk499490** [comment, marked as answer]
無難に、データベースをダンプ→Misskey再構築→データベース流し込み、で復帰しました。
再構築は[公式の手順](https://misskey-hub.net/ja/docs/for-admin/install/guides/docker/)で行い、`sudo docker compose run --rm web pnpm run init`の前に[データベース流し込み](https://qiita.com/nexryai/items/a1a4d07fce338796535b)を行いました。
